src/storage/src/hummock/compactor/shared_buffer_compact.rs

```diff
@@ -40,14 +42,20 @@ use crate::monitor::StoreLocalStatistic;
 pub async fn compact(
     context: Arc<Context>,
     payload: UploadTaskPayload,
+    compaction_group_index: Arc<HashMap<TableId, CompactionGroupId>>,
 ) -> HummockResult<Vec<(CompactionGroupId, SstableInfo)>> {
     let mut grouped_payload: HashMap<CompactionGroupId, UploadTaskPayload> = HashMap::new();
     for uncommitted_list in payload {
         let mut next_inner = HashSet::new();
         for uncommitted in uncommitted_list {
             let compaction_group_id = match &uncommitted {
                 UncommittedData::Sst((compaction_group_id, _)) => *compaction_group_id,
-                UncommittedData::Batch(batch) => batch.compaction_group_id(),
+                UncommittedData::Batch(batch) => {
+                    match compaction_group_index.get(&batch.table_id) {
+                        None => StaticCompactionGroupId::StateDefault as CompactionGroupId,
```

Comment:
we need some comment to explain the behavior of server about how to handler the StaticCompactionGroupId ?

Reply:
> we need some comment to explain the behavior of server about how to handler the StaticCompactionGroupId ?

The reason here is that
this table is very possible to belong to `StateDefault`